hx_requests/hx_requests.py

```diff
@@ -83,6 +83,7 @@ class BaseHXRequest:
     show_messages: bool = True
     get_views_context: bool = True
     kwargs_as_context: bool = True
+    opt_out_get_context: bool = False
```

Comment:
instead of this...what about another function for get_GET_context and get_context_data calls both. this way like in your case where if have context only want on get, instead of putting in get_context_data you would put in get_GET_context and then no need for new attribute